lib/osf-components/addon/components/file-browser/add-new/template.hbs

```diff
@@ -0,0 +1,37 @@
+{{#let (unique-id 'upload-button-id') as |uploadButtonId|}}
+    <ResponsiveDropdown
+        @renderInPlace={{true}}
+        @buttonStyling={{true}}
+        @horizontalPosition='left'
+        as |dropdown|
+    >
+        <BsButton
```

Comment:
My preference would be to use our <Button> component here as well, but if there are some issues with styling, BsButton is ok too